### ISSUE TITLE
Inference results

### ISSUE THREAD

**Issue by ranhaogong** [YujiaLiu76/point2building#4]
When I used the provided pretrain model to inference data from testset, I got many bad results. There were holes and unfilled surfaces on the predicted results, and could not be watertight.
Also, I trained the model from scratch, the results were also bad.
What's the possible reason, looking forward to your suggestions!
![image](https://github.com/user-attachments/assets/ed615b2f-a95e-413b-92c0-2359959d29d2)
![image](https://github.com/user-attachments/assets/f50f5042-c75a-4920-8067-28715950d352)
![image](https://github.com/user-attachments/assets/e6cd476e-7247-4420-bc89-278bc8904b55)
![image](https://github.com/user-attachments/assets/fef2aebf-7cb6-4670-a5a2-1794dabf3be4)
![image](https://github.com/user-attachments/assets/37d93b30-e85e-48db-8fd7-7a79198808e4)

**Comment by DAGDADA** [comment]
Hi! I’ve had the same problem. Did you get it sorted out later?

**Comment by DAGDADA** [comment]
> When I used the provided pretrain model to inference data from testset, I got many bad results. There were holes and unfilled surfaces on the predicted results, and could not be watertight. Also, I trained the model from scratch, the results were also bad. What's the possible reason, looking forward to your suggestions! ![image](https://github.com/user-attachments/assets/ed615b2f-a95e-413b-92c0-2359959d29d2) ![image](https://github.com/user-attachments/assets/f50f5042-c75a-4920-8067-28715950d352) ![image](https://github.com/user-attachments/assets/e6cd476e-7247-4420-bc89-278bc8904b55) ![image](https://github.com/user-attachments/assets/fef2aebf-7cb6-4670-a5a2-1794dabf3be4) ![image](https://github.com/user-attachments/assets/37d93b30-e85e-48db-8fd7-7a79198808e4)



> When I used the provided pretrain model to inference data from testset, I got many bad results. There were holes and unfilled surfaces on the predicted results, and could not be watertight. Also, I trained the model from scratch, the results were also bad. What's the possible reason, looking forward to your suggestions! ![image](https://github.com/user-attachments/assets/ed615b2f-a95e-413b-92c0-2359959d29d2) ![image](https://github.com/user-attachments/assets/f50f5042-c75a-4920-8067-28715950d352) ![image](https://github.com/user-attachments/assets/e6cd476e-7247-4420-bc89-278bc8904b55) ![image](https://github.com/user-attachments/assets/fef2aebf-7cb6-4670-a5a2-1794dabf3be4) ![image](https://github.com/user-attachments/assets/37d93b30-e85e-48db-8fd7-7a79198808e4)

Hi! I’ve had the same problem. Did you get it sorted out later?

**Comment by ranhaogong** [comment]
No, I gave up this code and the author seems never reply to anyone. @DAGDADA
